app/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
 
 # Create your models here.
+'''
+1. We can have multiple Markets
+2. Each market can have multiple Shops
+3. Each Shop can have multiple Products
+4. Each Product can have multiple Brands, Price
+5. Each Product can have different Rating according to Brand
+'''
+
+class Market(models.Model):                  #market , shop, product
+    name=models.CharField(max_length=50)
+
+class Shop(models.Model):
+    name=models.CharField(max_length=50)
+    market=models.ForeignKey(Market, on_delete=models.CASCADE)
+
+
+class Brand(models.Model):
+    name=models.CharField(max_length=50)
+
+class Product(models.Model):
+    name=models.CharField(max_length=50)
```

Comment:
A product can be available on multiple shops and Price of a product will be according to the Brand.